<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewTextureAtlasDialog</class>
 <widget class="QDialog" name="NewTextureAtlasDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="descriptionLabel">
     <property name="text">
      <string>To create a new texture atlas tab, fill in the information below:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="newTabNameLabel">
       <property name="text">
        <string>New Tab Name: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="newTabNameLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="textureAtlasPropertiesLabel">
     <property name="text">
      <string>Specify the texture atlas properties associated with the new tab:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="textureAltasWidthLabel">
       <property name="text">
        <string>New Texture Atlas Width:  </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="texutreAtlasWidthLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="textureAtlasHeightLabel">
       <property name="text">
        <string>New Texture Atlas Height: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="textureAtlasHeightLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="textureAtlasFormatLabel">
       <property name="text">
        <string>Format of Texture Atlas:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="textureFormatComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="textureAtlasFormatNoteLabel">
     <property name="text">
      <string>Note: The format of the atlas will determine what the atlas is exported as.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewTextureAtlasDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewTextureAtlasDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
